lib/sentry/client.ex

```diff
@@ -8,10 +8,13 @@ defmodule Sentry.Client do
     unquote(@sentry_client "sentry-elixir/#{Mix.Project.config[:version]}")
   end
 
+  @moduledoc """
+    Provides basic HTTP client request and response handling for the Sentry API. 
+  """
+
   @spec send_event(%Event{}) :: {:ok, String.t} | :error
   def send_event(%Event{} = event) do
-    {endpoint, public_key, secret_key} = Application.fetch_env!(:sentry, :dsn)
-                                          |> parse_dsn!
+    {endpoint, public_key, secret_key} = parse_dsn!(Application.fetch_env!(:sentry, :dsn))
```

Comment:
I guess I don't understand credo's issue with this.  piping the result of a function into another function is readable enough?